faq/devcuda.inc

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/////////////////////////////////////////////////////////////////////////
+
+$q[] = "What are some guidelines for developing CUDA-aware Open MPI applications?";
+
+$anchor[] = "mpi-cuda-dev";
+
+$a[] = "Developing CUDA-aware applications is a complex topic, and beyond
+the scope of this document. CUDA-aware applications often have to take 
+machine-specific considerations into account, including the number of GPUs
+installed on each node and how the GPUs are connected to the CPUs and to each
+other. Often, when using a particular transport layer (such as OPA/PSM2)
+there will be run-time decisions to make about which CPU cores will be used
+with which GPUs.
+
+A good place to start is the <a href=\"https://docs.nvidia.com/cuda/\">nVidia CUDA Toolkit Documentation</a>, including the <a href=\"https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html\">Programming Guide</a>
+and the <a href=\"https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html\">Best Practices Guide</a>.
+For examples of how to write CUDA-aware MPI applications, the <a href=\"https://github.com/NVIDIA-developer-blog/code-samples/tree/master/posts/cuda-aware-mpi-example\">nVidia developers blog</a>
+offers examples and the <a href=\"http://mvapich.cse.ohio-state.edu/benchmarks/\">OSU
```

Comment:
http?  Not https?

Reply:
> http? Not https?

If you're referring to the OSU page, Yup - believe it or not the site does not support https....